regulations/views/section.py

```diff
@@ -15,21 +16,22 @@
 
 class SectionView(SidebarContextMixin, TemplateView):
 
-    url_path_view = 'chrome_section_view'
-
     template_name = 'regulations/section.html'
 
     sectional_links = True
 
+    client = api_reader.ApiReader()
+
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
 
         # getting url info (label and version)
         # answering the question: what are we looking at?
-        label_id = context['label_id']
         version = context['version']
-        label_id_list = label_id.split('-')
-        reg_part = label_id_list[0]
+        reg_part = context["part"]
+        reg_section = context["section"]
+        label_id = f"{reg_part}-{reg_section}"
+        label_id_list = [reg_part, reg_section]
```

Comment:
This is only used for `label_to_text`. It's worth examining how that works and if it's needed.